less/icons.less

```diff
@@ -127,3 +127,15 @@
     color: @tyrell-foreground;
   }
 }
+
+.icon-gold {
+  height: 12px;
+  width: 12px;
+  margin-top: -4px;
+}
+
+.icon-card {
+  height: 16px;
+  width: 12px;
+  margin-top: -4px;
```

Comment:
Can this positioning be achieved with `vertical-align` instead? (The answer is often no, but I think it's worth a try vs finicky margin adjustments)

It might also be marginally useful to have the icons scale based on current font size (use `em` units) instead of fixed pixels.

Reply:
Vertical-align only applies to block styled elements and this is in a span. It could be done with flex box but I’m not sure it’s worth it

Reply:
Vertical align should apply between inline elements, including inline-block elements (which this img should be). https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align

Like I said, though, it may not give you the exact alignment you're going for, but may be worth a look.